tutor/commands/upgrade/k8s.py

```diff
@@ -171,13 +183,23 @@ def upgrade_mongodb(
             "responsibility to upgrade your MongoDb instance to {to_docker_version}."
         )
         return
+    mongo_version = int(to_docker_version[0])
```

Comment:
This makes a lot of shared code between compose.py and k8s.pyu. Can we move it to the commands/upgrade/common.py module?